Library/ViewModels/ActivityErroredBackingsTopCellViewModel.swift

```diff
@@ -0,0 +1,33 @@
+import KsApi
+import ReactiveSwift
+
+public protocol ActivityErroredBackingsTopCellInputs {
+  func configure(with value: [GraphBacking])
+}
+
+public protocol ActivityErroredBackingsTopCellOutputs {
+  var erroredBackings: Signal<[GraphBacking], Never> { get }
+}
+
+public protocol ActivityErroredBackingsTopCellViewModelType {
+  var inputs: ActivityErroredBackingsTopCellInputs { get }
+  var outputs: ActivityErroredBackingsTopCellOutputs { get }
+}
+
+public final class ActivityErroredBackingsTopCellViewModel: ActivityErroredBackingsTopCellViewModelType,
+  ActivityErroredBackingsTopCellInputs, ActivityErroredBackingsTopCellOutputs {
+  public init() {
+    self.erroredBackings = self.backingsSignal
+      .skipRepeats()
```

Comment:
In this case, we reload the table view when we fetch erroredBackings and when after fetching activities. The fetchBackings response usually arrives first, so `backingsSignal` emits the first time (when the data source configures the `ActivityErroredBackingsCell`). Once we finish fetching activities, we reload the tableview again, and because we already have the `ActivityErroredBackingsCell` configured, the `backingsSignal` would emits again.